sdks/python/apache_beam/examples/complete/estimate_pi_test.py

```diff
@@ -47,5 +47,4 @@ def test_basics(self):
 
 
 if __name__ == '__main__':
-  logging.getLogger().setLevel(logging.INFO)
```

Comment:
There were some `logging.info()` calls that was removed earlier. importing logging module become unnecessary at that point but I forgot to remove this call. We can keep this, to see info data from the sdk. What would you prefer?